docs/current/162770-faq.md

```diff
@@ -12,6 +12,45 @@ We currently offer technical previews of a [Go SDK](/sdk/go) and a [Python SDK](
 
 Dagger SDKs use your existing Docker credentials and do not require separate authentication. Simply execute `docker login` against your container registry on the host where your Dagger pipelines are running.
 
+### What is the Dagger Engine?
+
+The Dagger Engine consists of two components: an API router and a runner.
+
+* The router serves API queries and dispatches individual operations to the runner.
+* The runner talks to an Open Container Inititaive (OCI) runtime to execute actual operations. This is basically a BuildKit daemon with some glue code.
+
+Currently, the router is on a client machine, and the runner is on a worker machine that runs the containers. This could be the same machine, but typically isn't.
```

Comment:
I would change this line to something which is accurate today:

```diff
- Currently, the router is on a client machine, and the runner is on a worker machine that runs the containers. This could be the same machine, but typically isn't.
+ Currently, both the router and the runner are executed on the client machine. In the near-term, we expect the runner to move to worker machines.
```